doculog/main.py

```diff
@@ -19,3 +24,41 @@ def generate_changelog():
 
     if log_path.exists():
         print(f"Saved changelog to {log_path}")
+
+
+parser = ArgumentParser(
+    prog="doculog",
+    description=f"Doculog v{__version__}",
+)
+
+# parser.add_argument("-cl", "--changelog", 
```

Comment:
Looks good. Can we remove the commented out code though please. We have the commit log so can always just revert when we want to re-add it